src/main/kotlin/ro/hibyte/betting/service/EventService.kt

```diff
@@ -50,8 +75,19 @@ class EventService(
         eventRepository.save(event)
     }
 
+    fun deleteEvent(eventId: Long) {
```

Comment:
maybe we should never delete events, but rather use a flag of "deactivated" - just an idea.

 but not part of the story, you can disregard this comment